.github/workflows/upload-to-r2.yml

```diff
@@ -0,0 +1,42 @@
+name: Upload Markdown to R2
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'content/**/*.md'   
+      - 'content/**/*.mdx'  
+  schedule:
```

Comment:
We don't need the schedule here since the `paths` is already capturing changes to the `content` folder.